rotkehlchen/globaldb/updates/types.py

```diff
@@ -0,0 +1,36 @@
+from enum import Enum, auto
+from typing import NamedTuple
+
+from rotkehlchen.assets.types import AssetType
+from rotkehlchen.types import Timestamp
+
+
+class ParsedAssetData(NamedTuple):
```

Comment:
We already have `AssetData` in assets/types.py. Why create yet another data structure here which is almost the same?

Reply:
It is moved from other file. 
![imagen](https://github.com/user-attachments/assets/3846b453-3221-40cd-afa8-c6d17e0ad37e)

We should create an issue to get rid of it